locale/en/foreman_remote_execution.po

```diff
@@ -1,19 +1,760 @@
+msgstr ""
+
+msgid "Repeat a maximum of N times"
+msgstr ""
+
+msgid "Perform no more executions after this time"
+msgstr ""
+
+msgid "Schedule the job to start at a later time"
+msgstr ""
+
+msgid "Schedule the job for a future time"
+msgstr ""
+
+msgid ""
+"Indicates that the action should be cancelled if it cannot be started before t"
+"his time."
+msgstr ""
+
+msgid "Override the description format from the template for this invocation only"
+msgstr ""
+
+msgid "Create a job invocation"
+msgstr ""
+
+msgid "Get output for a host"
+msgstr ""
+
+msgid "Host with id '%{id}' was not found"
+msgstr ""
+
+msgid "Template with id '%{id}' was not found"
+msgstr ""
+
+msgid "List job templates"
+msgstr ""
+
+msgid "List job templates per location"
+msgstr ""
+
+msgid "List job templates per organization"
+msgstr ""
+
+msgid "Show job template details"
+msgstr ""
+
+msgid "Template name"
+msgstr ""
+
+msgid "Job category"
+msgstr ""
+
+msgid ""
+"This template is used to generate the description. Input values can be used us"
+"ing the syntax %{package}. You may also include the job category and template "
+"name using %{job_category} and %{template_name}."
+msgstr ""
+
+msgid "Provider type"
+msgstr ""
+
+msgid "Whether or not the template is locked for editing"
+msgstr ""
+
+msgid "Effective user options"
+msgstr ""
+
+msgid "What user should be used to run the script (using sudo-like mechanisms)"
+msgstr ""
+
+msgid ""
+"Whether it should be allowed to override the effective user from the invocatio"
+"n form."
+msgstr ""
+
+msgid "Whether the current user login should be used as the effective user"
+msgstr ""
+
+msgid "Create a job template"
+msgstr ""
+
+msgid "Update a job template"
+msgstr ""
+
+msgid "Template version"
+msgstr ""
+
+msgid "Delete a job template"
+msgstr ""
+
+msgid "Clone a provision template"
+msgstr ""
+
+msgid "List template inputs"
+msgstr ""
+
+msgid "Show template input details"
+msgstr ""
+
+msgid "Input name"
+msgstr ""
+
+msgid "Input description"
+msgstr ""
+
+msgid "Input is required"
+msgstr ""
+
+msgid "Input is advanced"
+msgstr ""
+
+msgid "Input type"
+msgstr ""
+
+msgid "Fact name, used when input type is fact"
+msgstr ""
+
+msgid "Variable name, used when input type is variable"
+msgstr ""
+
+msgid "Puppet class name, used when input type is puppet_parameter"
+msgstr ""
+
+msgid "Puppet parameter name, used when input type is puppet_parameter"
+msgstr ""
+
+msgid "Selectable values for user inputs"
+msgstr ""
+
+msgid "Create a template input"
+msgstr ""
+
+msgid "Delete a template input"
+msgstr ""
+
+msgid "Update a template input"
+msgstr ""
+
+msgid ", note that you must save template input changes before you try to preview it."
+msgstr ""
+
+msgid "Run Job"
+msgstr ""
+
+msgid "Run"
+msgstr ""
+
+msgid "Success"
+msgstr ""
+
+msgid "Failed"
+msgstr ""
+
+msgid "Pending"
+msgstr ""
+
+msgid "Cancelled"
+msgstr ""
+
+msgid "queued"
+msgstr ""
+
+msgid "running %s%"
```

Comment:
I'm not sure what to suggest here, but will say that it's highly likely to get mistranslated on occasion.  While you can't just append the percent sign and to localise it properly means it should be included in the string, you might want to rewrite it as %{percent}% and/or add a TRANSLATORS command in the source code immediately before to say it's a percentage and should contain the % sign.  Hopefully then it won't be deleted.